.github/workflows/_create-release-pr.yml

```diff
@@ -56,14 +56,13 @@ jobs:
 
     - name: Create RC branch
       env:
+        RELEASE_BRANCH: ${{ steps.vars.outputs.release-branch }}
         RC_BRANCH: ${{ steps.vars.outputs.rc-branch }}
         TITLE: ${{ steps.vars.outputs.title }}
       run: |
-        git checkout -b "${RC_BRANCH}"
+        git switch -c "${RC_BRANCH}"
 
-        # create an empty commit to distinguish workflow runs
-        # from other possible releases from the same commit
-        git commit --allow-empty -m "${TITLE}"
+        git merge --commit -m "${TITLE}" "${RELEASE_BRANCH}" -s ours
```

Comment:
`-s ours` means anything on the release branch is fully ignored. From our conversation in https://neondb.slack.com/archives/C033RQ5SPDH/p1740653236436569?thread_ts=1740615531.219759&cid=C033RQ5SPDH, this is what we want, but we should probably confirm that there's consensus on this.